### PR TITLE
feat: hide zone/authn in listeners and remove listeners's authn api

### PR DESCRIPTION
**Pull request by zhongwencool**
Fixes 
https://emqx.atlassian.net/browse/EMQX-10286
https://emqx.atlassian.net/browse/EMQX-10250

### <samp>🤖 Generated by Copilot at 7668753</samp>

This pull request removes the deprecated zone and listener authentication features from emqx, and simplifies the authentication and listener management logic. It also updates the user interface, the API, and the test cases to reflect the changes. It affects the files `emqx_schema.erl`, `emqx_listeners.erl`, `emqx_authn_api.erl`, `emqx_authn_user_import_api.erl`, `emqx_mgmt_api_configs.erl`, `emqx_mgmt_api_listeners.erl`, `emqx_authn_api_SUITE.erl`, and `emqx_mgmt_api_configs_SUITE.erl`.

## PR Checklist
Please convert it to a draft if any of the following conditions are not met. Reviewers may skip over until all the items are checked:

- [ ] Added tests for the changes
- [ ] Changed lines covered in coverage report
- [ ] Change log has been added to `changes/{ce,ee}/(feat|perf|fix)-<PR-id>.en.md` files
- [ ] For internal contributor: there is a jira ticket to track this change
- [ ] If there should be document changes, a PR to emqx-docs.git is sent, or a jira ticket is created to follow up
- [ ] Schema changes are backward compatible

## Checklist for CI (.github/workflows) changes

- [ ] If changed package build workflow, pass [this action](https://github.com/emqx/emqx/actions/workflows/build_packages.yaml) (manual trigger)
- [ ] Change log has been added to `changes/` dir for user-facing artifacts update

### CODE REVIEW

File: apps/emqx_conf/src/emqx_conf_cli.erl [hunk 1]
```diff
@@ -151,7 +151,8 @@ status() ->
     emqx_ctl:print("-----------------------------------------------\n").
 
 print_keys(Config) ->
-    print(lists:sort(maps:keys(Config))).
+    Keys = lists:sort(maps:keys(Config)),
+    emqx_ctl:print("~1p~n", [[binary_to_existing_atom(K) || K <- Keys]]).
```
Comment (zhongwencool) [inline apps/emqx_conf/src/emqx_conf_cli.erl:155]: "1p" print each key one line

File: apps/emqx_management/src/emqx_mgmt_api_configs.erl [hunk 2]
```diff
@@ -43,9 +43,8 @@
     <<"alarm">>,
     <<"sys_topics">>,
     <<"sysmon">>,
-    <<"log">>,
-    <<"persistent_session_store">>,
-    <<"zones">>
```
Comment (zhongwencool) [inline apps/emqx_management/src/emqx_mgmt_api_configs.erl:48]: Also remove persistent_session_store. It's root key is hidden since 5.1.0

**Review by HJianBo — APPROVED** [review]

File: apps/emqx/src/emqx_schema.erl [hunk 3]
```diff
@@ -3409,7 +3410,7 @@ mqtt_general() ->
             )},
         {"server_keepalive",
             sc(
-                hoconsc:union([integer(), disabled]),
+                hoconsc:union([pos_integer(), disabled]),
```
Comment (JimMoen) [inline apps/emqx/src/emqx_schema.erl:3413]: should nvere be `0` ?

**Comment by JimMoen** [comment]
other related files:

https://github.com/emqx/emqx/blob/release-51/apps/emqx_authn/test/emqx_authn_enable_flag_SUITE.erl#L78

https://github.com/emqx/emqx/blob/release-51/apps/emqx_management/src/emqx_mgmt_api_clients.erl#L66

**Comment by yanzhiemq** [comment]
_

**Comment by yanzhiemq** [comment]
_

**Pull request merged**